apps/src/templates/rubrics/RubricSettings.jsx

```diff
@@ -208,13 +187,13 @@ export default function RubricSettings({
               // we can't fetch the csrf token from the DOM because CSRF protection
               // is disabled on script level pages.
               setCsrfToken(data.csrfToken);
-              setUnevaluatedCount(data.attemptedUnevaluatedCount);
+              setEvaluatedCount(data.lastAttemptEvaluatedCount);
```

Comment:
Why is this changed here, but not in the initial useEffect? Is EvaluatedCount used anywhere? UnevaluatedCount is used for the ready status string, is it fine for that to not be updated?

Reply:
Ok, I thought there were cases where it could be updated on polling, but it looks like polling is done only after the click. Looks good to me!